### DISCUSSION TITLE
Custom Image SSTV

### DISCUSSION THREAD

**Discussion by samuellbs** [jgromes/RadioLib#1062] [Q&A]
Hey everyone,

I've been playing with SSTV lately and i'm trying to make a custom image to transmit but i'm not understand how to do it. Could someone explain me how it works and give me a example?

**Comment by jgromes** [comment]
First you need your picture in raw 24-bit colors - RadioLib SSTTV driver expects each pixel to be a 32-bit number. You also need to make sure that the picture has the correct size for the SSTV mode you want to use (e.g. 320x256 pixels for Scottie/Martin/Wrasse modes). Then, you need to pass that picture line-by-line to `SSTV::sendLine`, just like the examples show.

You will also most likely need some storage for that picture (like an SD card or external flash), since in raw, it's going to be 320x256x4 bytes, or 320 kB - you have not said what platform you are using but there are not many that can support that sort of memory footprint.
